src/doc/imagebuf.rst

```diff
@@ -474,37 +445,14 @@ Strategy 2: Template your iterating functions based on buffer type
 Consider the following alternate version of the `make_black` function
 from Section `Example: Set all pixels in a region to black`_ ::
```

Comment:
Another spot where the `::` needs to change to `:` because we replaced intended code itself with `.. tabs::` commands.